ouroboros-consensus/src/Ouroboros/Consensus/HardFork/Combinator/Serialisation/SerialiseNodeToClient.hs

```diff
@@ -189,6 +188,10 @@ encodeQueryHardFork vHfc = \case
         Enc.encodeListLen 1
       , Enc.encodeWord8 1
       ]
+    Some GetLedgerCfg -> mconcat [
```

Comment:
Above this you can see another example of the explicit versioning of these queries.

Reply:
updated